Tests/test_file_xvthumb.py

```diff
@@ -1,13 +1,35 @@
-from helper import unittest, PillowTestCase
+from helper import hopper, unittest, PillowTestCase
 
-from PIL import XVThumbImagePlugin
+from PIL import Image, XVThumbImagePlugin
+
+TEST_FILE = "Tests/images/hopper.p7"
 
 
 class TestFileXVThumb(PillowTestCase):
 
+    def test_open(self):
+        # Act
+        im = Image.open(TEST_FILE)
+
+        # Assert
+        self.assertEqual(im.format, "XVThumb")
+        self.assert_image_similar(im, hopper("P"), 49)
```

Comment:
It's the palette at the top of XVImagePlugin. We should be able to use Image.quantize with (a version of) that palette to get something that's very close to an XV image. 

((and incidentally, that's probably a way to save it as well. Quantize to that palette, then call getdata and write that to the file))